config/config-windows.yaml

```diff
@@ -263,6 +291,63 @@ processors:
           # change data type from int64 -> double
           - action: toggle_scalar_data_type
 
+  # filter to include only agent metrics supported by cloud monitoring
+  filter/agent:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - otelcol_process_uptime
+          - otelcol_process_memory_rss
+          - otelcol_grpc_io_client_completed_rpcs
+          - otelcol_otelcol_googlecloudmonitoring_point_count
+
+  # convert from opentelemetry metric formats to cloud monitoring formats
+  metricstransform/agent:
+    transforms:
+      # otelcol_process_uptime -> agent/uptime
+      - metric_name: otelcol_process_uptime
+        action: update
+        new_name: agent/uptime
+        operations:
+          # change data type from double -> int64
+          - action: toggle_scalar_data_type
+          # add version label
+          - action: add_label
+            new_label: version
+            new_value: Google Cloud Operations OpenTelemetry Collector {{version}}
+      # otelcol_process_memory_rss -> agent/memory_usage
+      - metric_name: otelcol_process_memory_rss
+        action: update
+        new_name: agent/memory_usage
+      # otelcol_grpc_io_client_completed_rpcs -> agent/api_request_count
+      - metric_name: otelcol_grpc_io_client_completed_rpcs
+        action: update
+        new_name: agent/api_request_count
+        operations:
+          # change data type from double -> int64
+          - action: toggle_scalar_data_type
+          # ignore any non "google.monitoring" RPCs
+        # TODO (there won't be any of these, but would be safer to delete to make this future proof):
```

Comment:
What prevents you from addressing this TODO in this PR?

Reply:
This feature doesn't exist yet 😄  This is more of a proposed config for the transform processor.

I'll update the comment to be more specific.